connection_test.go

```diff
@@ -748,3 +748,19 @@ func TestConnectTimeout(t *testing.T) {
 		close(testComplete)
 	})
 }
+
+func TestConnectionProcessPrefixes(t *testing.T) {
+	opts := testutils.NewOpts().NoRelay().SetProcessName("nodejs-hyperbahn")
+	testutils.WithTestServer(t, opts, func(ts *testutils.TestServer) {
+		clientOpts := testutils.NewOpts().NoRelay().SetProcessPrefixes("nod", "nodejs-hyperbahn", "", "hyperbahn")
+		client := ts.NewClient(clientOpts)
+
+		ctx, cancel := NewContextBuilder(testutils.Timeout(100 * time.Millisecond)).Build()
```

Comment:
Don't need the builder here since we're not setting any custom context properties. Since we don't rely on timeout in the test, we can do what most of the tests do, `ctx, cancel := NewContext(time.Second)`